sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -205,6 +205,8 @@ case class AdaptiveSparkPlanExec(
 
   def executedPlan: SparkPlan = currentPhysicalPlan
 
+  def isFinalized: Boolean = isFinalPlan
```

Comment:
Can we do `private var _isFinalPlan` and `def isFinalPlan` instead?

Reply:
sure, addressed